languages/rust/exercises/concept/tuples/.docs/instructions.md

```diff
@@ -0,0 +1,56 @@
+# Instructions
+
+In this exercise you'll explore tuples in Rust by building a kitchen calculator. Imagine you are following a sensible gingersnap cookie recipe but it uses American units for measuring ingredients. You are most familiar with metric units. Being an intrepid programmer, you want to automate that work by writing a kitchen calculator.
+
+Use the following chart in your program:
+
+| Unit to convert | volume | in millilitres (mL) |
+| --------------- | ------ | ------------------- |
+| mL              | 1      | 1                   |
+| US cup          | 1      | 240                 |
+| US fluid ounce  | 1      | 30                  |
+| US teaspoon     | 1      | 5                   |
+| US tablespoon   | 1      | 15                  |
+
+## 1. Extract the numeric component from a tuple describing volume
```

Comment:
Task headings are usually written as relatively high-level, non-technical descriptions. Maybe something like this:

```suggestion
## 1. Extract the volume from a unit/volume pair
```

Or something similar. This also applies to the other headings.